src/HasProfilePhoto.php

```diff
@@ -56,19 +57,55 @@ public function getProfilePhotoUrlAttribute()
     {
         return $this->profile_photo_path
                     ? Storage::disk($this->profilePhotoDisk())->url($this->profile_photo_path)
-                    : $this->defaultProfilePhotoUrl();
+                    : '';
     }
 
     /**
      * Get the default profile photo URL if no profile photo has been uploaded.
+     * Only return the default profile photo url if Jetstream is configured to manage profile photos.
+     * This reduces unnecessary web requests.
      *
      * @return string
      */
     protected function defaultProfilePhotoUrl()
     {
+        if (! Features::managesProfilePhotos()) {
+            return '';
```

Comment:
It's also a breaking change.

Reply:
why is everything returning empty string?